pnpm-lock.yaml

```diff
@@ -26662,6 +26728,37 @@ packages:
       - zod
     dev: false
 
+  /wagmi@1.3.10(encoding@0.1.13)(react-dom@18.2.0)(react@18.2.0)(typescript@5.1.3)(viem@1.2.9):
```

Comment:
pnpm version got little tricky there. duplicated wagmi will throw error in runtime
please stick with pnpm@8.6.0 for now

Reply:
Done